projects/c_api/Readme.md

```diff
@@ -0,0 +1,49 @@
+# OpenDR C API
+
+## Description
+
+This module contains a C API that can be used for performing inference on models trained using the Python API of OpenDR. 
+Therefore, to use the C API you should first use the Python API to train a model and then export it to ONNX format using the `optimize()` method.
+
+## Setup
+Before using the C API you should have the appropriate version of ONNX runtime installed. 
+You can use the provided script (`install_deps.sh`) to install it, along with a number of other required dependencies.
+Alternatively, you can also use the supplied Makefile that takes care both of the installation and compilation.
+Please note that only Ubuntu 20.04 is currently officially supported.
+
+In order to build the library you should:
+
+0. Install the required binary dependencies:
+
+```make install_dependencies```
+
+1. Build the static library
+
+```make liboperdr```
+
+2. (Optionally) Download the necessary resources and verify that everything works as expected:
+
+```
+make download
+make runtests
+```
+After building the API you will find the static library (`libopendr.a`) under the `lib` folder.
```

Comment:
Isn't better to create a dynamic/shared library instead of a static library?

Reply:
Switched into using a shared library.